docs/ref/contrib/postgres/operations.txt

```diff
@@ -59,3 +59,35 @@ the ``django.contrib.postgres.operations`` module.
 .. class:: UnaccentExtension()
 
     Installs the ``unaccent`` extension.
+
+``Create extension using migrations``
+=====================================
+
+You can create a PostgreSQL Extension in your database using a migration file.
+The following example will cover creating of hstore extension,
+but same usage principles apply for all other extensions, mentioned above.
+
+1. Add ``'django.contrib.postgres'`` in your :setting:`INSTALLED_APPS`.
```

Comment:
There is a big difference in length of the first and second point. Better not to use a numbered list. Perhaps do the first point as a separate paragraph like this?

    Prerequisite: ``'django.contrib.postgres'`` must be in your :setting:`INSTALLED_APPS` setting.